public/app/features/plugins/admin/state/actions.ts

```diff
@@ -63,6 +64,9 @@ export const install = createAsyncThunk(
     try {
       await installPlugin(id, version);
       await updatePanels();
+      if (isUpdating) {
```

Comment:
Any tests that need to change/update because of this change?

Reply:
I think so. Let me have a look.